### ISSUE TITLE
Querying not usable Stamp returns `400`

### ISSUE THREAD

**Issue by AuHau** [ethersphere/bee#2373]
#### Summary
When querying stamp by ID that was newly created and is still not usable the HTTP request fails with 400 instead of returning the Stamp object with `usable: false`.

This happens because there is a check in [`GetStampIssuer`](https://github.com/ethersphere/bee/blob/5aa39499e52563071cee1e559a533841d9c20768/pkg/postage/service.go#L132) for usability which fails because of it.

#### Steps to reproduce

 1. Create stamp
 2. Query the stamp by ID before it becomes usable

#### Expected behavior

Returned object with `usable: false`

#### Actual behavior

HTTP 400 with
```
time="2021-07-29T07:14:50Z" level=debug msg="get stamp issuer: get issuer: not usable"
time="2021-07-29T07:14:50Z" level=error msg="get stamp issuer: get issuer"
time="2021-07-29T07:14:50Z" level=info msg="debug api access" duration=0.0004778 ip=172.18.0.1 method=GET proto=HTTP/1.1 size=44 status=400 uri=/stamps/b75f5afe387b19057aed57f68f3704a0920b0b0bfab856bc905e6f76e327d61d user-agent=curl/7.64.1
```

**Comment by AuHau** [comment]
Would suggest to cover these cases with tests...

**Comment by mrekucci** [comment]
Fixed in #2375
